src/integration/integration_suite_test.go

```diff
@@ -451,39 +457,65 @@ func startFakeCCNOAAUAA(instanceCount int) {
 
 }
 
-func fakeMetrics(appId string, memoryValue uint64) {
+func fakeMetricsPolling(appId string, memoryValue uint64) {
 
 	eLock = &sync.Mutex{}
-	fakeCCNOAAUAA.RouteToHandler("GET", ccNOAAUAARegPath,
+	fakeCCNOAAUAA.RouteToHandler("GET", noaaPollingRegPath,
 		func(rw http.ResponseWriter, r *http.Request) {
-			eLock.Lock()
```

Comment:
this piece of code is used to test token expire case, we have done it in unit test.. we don't need to test every cases in  integration test ...